scripts/CMakeLists.txt

```diff
@@ -11,5 +11,6 @@ endfunction()
 install_scripts (jbdiagnose.sh plotjbbal.py plotjbdiag.py)
 install_scripts (varbcdiagnose.sh plotvarbcpred.py)
 install_scripts (rundiacov.sh diag_diacov.sh plotdiacov.py)
+install_scripts (runfestat.sh)
```

Comment:
I haven't added the Python linking script to the installation yet. Not yet tested.